openmdao/utils/concurrent.py

```diff
@@ -194,6 +194,10 @@ def concurrent_eval(func, cases, comm, allgather=False):
     allgather : bool(False)
         If True, the results will be allgathered to all procs in the comm.
         Otherwise, results will be gathered to rank 0 only.
+    model_mpi : None or tuple
+        If the function in func runs in parallel, then this will contain a tuple with the
+        total number of population points to evaluate concurrently, and the color of the point
+        to evalutate on this rank.
```

Comment:
Not quite clear on this, but couldn't you just pass in the `model_comm` as `comm` (instead of the problem/driver comm) and you wouldn't need this tuple and associated logic?
The description of the tuple also seems pretty specific to the GA in this general purpose function...

Reply:
No, I need driver.comm for the gathers.  But, I will change the wording in the docstring.

Reply:
oh right..  i was thinking of it backwards